controllers/fetchdata_controller.go

```diff
@@ -140,23 +143,23 @@ func NewCronJob(m *curatorv1alpha1.FetchData, scheme *runtime.Scheme) *batchv1.C
 										},
 										{
 											Name:  "DATABASE_NAME",
-											Value: m.Spec.DatabaseName,
+											Value: db.Config().Database,
```

Comment:
not blocking: This is a problem with the previous implementation, but I think we need to also update this implementation to avoid leaking sensitive information as environment variables for the generated CronJob resource as we rely on cluster admins setting up the requisite RBAC to avoid a situation where anyone in the cluster can view this CronJob output and pull out the postgres credentials.